vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -277,6 +278,7 @@ class LEDGER_EXPORT Ledger {
       const uint64_t from_timestamp_seconds,
       const uint64_t to_timestamp_seconds,
       ledger::ConfirmationsHistoryCallback callback) = 0;
+  virtual void GetRewardsInternalsInfo(ledger::RewardsInternalsInfo* info) = 0;
```

Comment:
it seems very odd to me that we would call this `RewardsInternals` inside ledger. I'm not going to block on it, but I don't think we shouldn't be tightly coupling Brave-specific UI to ledger methods. Isn't this data already available from other ledger datatypes?